### PR TITLE
Rating component

### PR DESCRIPTION
**Pull request by rutu14**
Preview: https://deploy-preview-11--friendly-kilby-44e498.netlify.app/documentation/rating.html

### CODE REVIEW

File: components/component.css [hunk 1]
```diff
@@ -48,6 +49,10 @@
     --disabled-inner: #E1E6F9;
```
Comment (zoyatanweer) [inline components/component.css:50]: remove spaces

**Review by zoyatanweer — APPROVED** [review]
good to merge. great UI

**Review by itsmebipul00 — APPROVED** [review]

File: documentation/css/prism-custom.css [hunk 2]
```diff
@@ -64,6 +64,18 @@ code[class*=language-], pre[class*=language-] {
     color: var(--code-js-string);
 }
 
+.code-toolbar .token.property{
+    color: #56b6c2;
+}
+
+.code-toolbar .token.selector{
+    color: #d19a66;
```
Comment (zoef-yt) [inline documentation/css/prism-custom.css]: use var for colours

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/component.css (hunk 1 above)
Comment (rutu14) [reply components/component.css:50]: Sure

File: documentation/css/prism-custom.css (hunk 2 above)
Comment (rutu14) [reply documentation/css/prism-custom.css]: Thanks for the reminder.

**Pull request merged**